### PR TITLE
create full web

### PR DESCRIPTION
**Pull request by TamaraDimov**
Create a full website for the Math magicians app, consisting of several pages and using the components you already created.
Create a website consisting of 3 pages: Home, Calculator, and Quote.
The general layout should match required wireframes.

### CODE REVIEW

File: src/components/style/Calculator.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  display: grid;
+} */
```
Comment (AmaduKamara) [inline src/components/style/Calculator.css]: - [ ] It's a good practice not to have commented unused code in our codebase. Kindly remove this unused commented code so your codebase will look clean and professional. 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @TamaraDimov,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Created 3 pages for the app as expected 👍🏽
✅ Navigation with react-router-dom is configured well as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] To help make the look of your app a little interactive or to have a little bit of padding and spacing from the texts to the edge of the body, you could add padding to the body of the main `index.css` file. 
When I added this `padding: 40px;` to the body styles of the main index.css file, it improves the spacing of texts from hitting the edges of all pages. Kindly consider adding that to help improve your app's look 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/style/Calculator.css (hunk 1 above)
Comment (TamaraDimov) [reply src/components/style/Calculator.css]: @AmaduKamara, thank you for the review and good explanation appreciate it! 🥇

**Review by DammyShittu — APPROVED** [review]
Hi @TamaraDimov,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**